### ISSUE TITLE
Cosmos azure helper failing with CosmosDB databases

### ISSUE THREAD

**Issue by richardasavie** [terraform-providers/terraform-provider-azurerm#7472]
I discover this while trying to import a mongodb database with the `terraform import` command, it fails to find the `databases` string from the ID. MongoDB databases in azure don't have that string in their ID, instead they have the `mongodbDatabases`.

When I run:
```
az cosmosdb mongodb database show --resource-group=<my-resource-group> --account-name=<my-db-account-name> --name my-database
```
It will return an id like the following:
```
/subscriptions/<my-subscription-id>/resourceGroups/<my-resource-group>/providers/Microsoft.DocumentDB/databaseAccounts/<my-db-account-name>/mongodbDatabases/my-database
```

If I try to import my dabase like:

```
terraform import resource.azurerm_cosmosdb_mongo_database.my-database /subscriptions/<my-subscription-id>/resourceGroups/<my-resource-group>/providers/Microsoft.DocumentDB/databaseAccounts/<my-db-account-name>/mongodbDatabases/my-database
```
Terraform throws an error:
```
Error: Error: Unable to parse Cosmos Database Resource ID: databases is missing from: /subscriptions/<my-subscription-id>/resourceGroups/<my-resource-group>/providers/Microsoft.DocumentDB/databaseAccounts/<my-db-account-name>/mongodbDatabases/my-database
```

But if I replace the mongodbDatabases in the id by databases, like so:
```
terraform import resource.azurerm_cosmosdb_mongo_database.my-database /subscriptions/<my-subscription-id>/resourceGroups/<my-resource-group>/providers/Microsoft.DocumentDB/databaseAccounts/<my-db-account-name>/databases/my-database
```
It all works.

The issue is in this function, because it is splitting by only the `databases` string:

https://github.com/terraform-providers/terraform-provider-azurerm/blob/6f80ed1c7f39d87af6882338be99a94f4599e75b/azurerm/helpers/azure/cosmos.go#L54

I am expecting the helper to work with both cosmosdb databases and mongodb databases.

Hope this helps

**Comment by yupwei68** [comment]
Hi @richardasavie , thanks for opening this issue. This is caused by different service version. Since service version 2019-08-01, cosmodb has accepted a new format of mongodb id like `/subscriptions/<my-subscription-id>/resourceGroups/<my-resource-group>/providers/Microsoft.DocumentDB/databaseAccounts/<my-db-account-name>/mongodbDatabases/my-database`. 
But the old one is still supported: `/subscriptions/<my-subscription-id>/resourceGroups/<my-resource-group>/providers/Microsoft.DocumentDB/databaseAccounts/<my-db-account-name>/apis/mongodb/databases/my-database`. 
As a temporary workaround, I suggest to use the elder mongodb id in Terraform. In long term, we'll support the new mongodb id when cosmo db is upgraded to a new version. But currently, I have not had a timeline for the upgrade.

**Comment by yupwei68** [comment]
Hi @richardasavie , sorry for the delay, currently cosmos db service version has been updated to 2020-04-01 and released in `2.20.0`.
Please refer to  https://github.com/terraform-providers/terraform-provider-azurerm/pull/7597.
And currently you could import your mongo db with url: `subscriptions/<my-subscription-id>/resourceGroups/<my-resource-group>/providers/Microsoft.DocumentDB/databaseAccounts/<my-db-account-name>/mongodbDatabases/my-database`.

**Comment by richardasavie** [comment]
Awesome folks! thanks for the notification @yupwei68 👍
